src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,67 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String NAME_SUPPLY = "supply";
+    private static final String NAME_BUY = "buy";
+    private static final String NAME_RESULT = "result";
+    private static final int DATA_NUMBER = 1;
+    public int supplyData = 0;
+    public int buyData = 0;
+
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(toFileName);
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(file))) {
+            writer.write("");
+            writer.flush();
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write changes", e);
+        }
+        getSupplyData(fromFileName);
+        for (String row : getReportData()) {
+            try (BufferedWriter writer = new BufferedWriter(new FileWriter(toFileName, true))) {
+                writer.write(row + System.lineSeparator());
+            } catch (IOException e) {
+                throw new RuntimeException("Can't write data to file", e);
+            }
+        }
+    }
+
+    public void getSupplyData(String fromFileName) {
```

Comment:
This method name does not match what the method does. Let's create a separate method that will read data from file and return some value